util/deploy.sh

```diff
@@ -51,7 +51,7 @@ if [[ "${APP_PATH}" == "webapp" ]]; then
 fi
 
 # Create a name for this version
-VERSION_BRANCH_NAME="$(echo ${BRANCH_NAME:-"$(git rev-parse --abbrev-ref HEAD)"} | tr /_ - | cut -c 1-28)"
+VERSION_BRANCH_NAME="$(echo ${BRANCH_NAME:-"$(git rev-parse --abbrev-ref HEAD)"} | tr [:upper:] [:lower:] | tr -c [:alnum:] - | cut -c 1-28)"
```

Comment:
This seems like it will replace all non-alphanumeric characters with one `-` for each - is that what we want? I'm not familiar with what branch names we tend to deal with.

Reply:
Yes, e.g. `Abc1_2.3-4` to `abc1-2-3-4`. Any concerns about that?

Reply:
That case looks fine. I was thinking of something like `Abc@#123`, which would just be a little weird (double-hyphen). But if that's rare/never happens, SGTM.

Reply:
Yeah hopefully no one does that...